### ISSUE TITLE
Physics Bones in ragdoll does not sync after reset

### ISSUE THREAD

**Issue by Corruptinator** [godotengine/godot#24934]
**Godot version:**
Godot 3.1 Beta 1

**OS/device including version:**
Windows 10

**Issue description:**
I was trying out the new Ragdoll feature for 3.1 by going through the ragdoll system article (https://godotengine.org/article/godot-ragdoll-system). I had no trouble getting the ragdoll to work but the what happened was very unusual:

![physic_bones](https://user-images.githubusercontent.com/5027750/51068865-31da7e00-15d9-11e9-87ff-6cba8866d655.gif)

I was expecting the ragdoll to reset back and forth seamlessly but the twitch shown in the example above shows that either it isn't synchronizing properly or there is a framerate issue.

**Steps to reproduce:**
Follow the tutorial to create a ragdoll (https://godotengine.org/article/godot-ragdoll-system) using the 3d Platformer provided through Github or Godot's Template Download at the Project Selection Window. Run the ragdoll simulation and see if you notice any twitching occuring using the physical_bones_start_simulation() and physical_bones_stop_simulation() within the skeleton node's script.

This is the code I used to test the ragdoll switch in the SKELETON NODE:

```
extends Skeleton

# Declare member variables here. Examples:
# var a = 2
# var b = "text"
var enable = false

# Called when the node enters the scene tree for the first time.
func _ready():
	pass # Replace with function body.

# Called every frame. 'delta' is the elapsed time since the previous frame.
#func _physics_process(delta):
#	pass

func _input(event):
	if event.is_action_pressed("ui_dead"):
		if enable == true:
			enable = false
		elif enable == false:
			enable = true
		else:
			pass
		_on_switch()

func _on_switch():
		if enable == true:
			physical_bones_start_simulation()
		elif enable == false:
			physical_bones_stop_simulation()
		else:
			pass
```

**Minimal reproduction project:**

[Platformer (Ragdoll Test).zip](https://github.com/godotengine/godot/files/2751681/Platformer.Ragdoll.Test.zip)

"ui_dead" is the action key "E" assigned to enable the ragdoll in case you need keyboard controls.

**Comment by Chaosus** [comment]
Hmm.. maybe its not a bug and incorrect usage or creation, cc @AndreaCatania

**Comment by Corruptinator** [comment]
I have no Idea, I still kept experimenting on the issue to find a resolution and I thought it was the Joint offset that needs to be calibrated to be in sync where the the other bones are located and rotated.

**Comment by Corruptinator** [comment]
Joint offset doesn't allow to be rotated or modified in the engine, even after using the code "joint_offset" in GDScript.

**Comment by AndreaCatania** [comment]
Cna you please try to use the 6DOF joint?

**Comment by Corruptinator** [comment]
Hang on. I'm going to download the 3.1 Beta 3 version and test it. I'll post results soon.

**Comment by Corruptinator** [comment]
6DOF causes the model to phase through the floor for some bizarre reason.

**Comment by akien-mga** [comment]
I can't seem to reproduce the bug in the current master branch (88a36e20cb), at least if the bug is the one-frame disappearance of the model when enabling ragdolls. It runs smoothly for me (Mageia 7 Linux, Nvidia GTX 670MX).

**Comment by Corruptinator** [comment]
Its been a while, but been playing around with 3.1 Beta 6 and the bones doesn't seem to have the one frame off-sync anymore. Maybe it got patched or something?

**Comment by akien-mga** [comment]
I guess it might have been fixed by #25402.

**Comment by Corruptinator** [comment]
Thanks! I'll test it out later and see how it looks.
